### PR TITLE
Release to Modrinth

### PR DESCRIPTION
**Pull request by NotMyFault**
Fixes https://github.com/IntellectualSites/FastAsyncWorldEdit/issues/1967

### CODE REVIEW

**Comment by NotMyFault** [comment]
Hey,

I'd like to ask @IntellectualSites/fastasyncworldedit-team members to sign up on https://modrinth.com/ if you didn't do so yet. I'll add everyone to the project.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -70,3 +70,8 @@ jobs:
         with:
           name: FastAsyncWorldEdit-Bukkit-SNAPSHOT
           path: worldedit-bukkit/build/libs/FastAsyncWorldEdit-Bukkit-*.jar
+      - name: Publish to Modrinth
+        if: ${{ runner.os == 'Linux' && env.STATUS == 'release' && github.event_name == 'push' && github.ref == 'refs/heads/main'}}
+        run: ./gradlew modrinth
+        env:
+          MODRINT_TOKEN: ${{ secrets.MODRINTH_TOKEN }}
```
Comment (PierreSchwang) [inline .github/workflows/build.yml:77]: typo (MODRINT[H]_TOKEN)

**Review by PierreSchwang — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (NotMyFault) [reply .github/workflows/build.yml:77]: Modrinth is spelled like that?

**Comment by PierreSchwang** [comment]
Have you yet tested the trask against the staging server?

**Comment by NotMyFault** [comment]
> Have you yet tested the task against the staging server?

The project has not been reviewed yet.

**Review by PierreSchwang — APPROVED** [review]

**Review by SirYwell — APPROVED** [review]

**Pull request merged**